### ISSUE TITLE
[Multiple Datasource] Help menu should include link to the documentation guide

### ISSUE THREAD

**Issue by joshuarrrr** [opensearch-project/OpenSearch-Dashboards#3707]
When reviewing https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3685, I noticed that the only in-application link to the multiple datasource documentation (https://opensearch.org/docs/latest/dashboards/discover/multi-data-sources/) is in the experimental callout, which will go away when the feature goes to GA in 2.7.0. 

That guide is a great resource - we should also add it to the help menu for the data source management plugin pages.

**Comment by joshuarrrr** [comment]
CC @zhongnansu - If we have time, I'd love to include this in the 2.7 scope.

**Comment by zhongnansu** [comment]
@shanilpa Please take a look, if it makes sense we could consider adding this to our roadmap.

**Comment by joshuarrrr** [comment]
ha, nvm, thanks @zhongnansu

**Comment by seraphjiang** [comment]
@zhongnansu is it still valid?

**Comment by zhongnansu** [comment]
> @zhongnansu is it still valid?

Yes, it's valid. After out of experimental, we no longer have a doc link to multiple datasource feature from the application.
